formal-resume/resume_english.tex

```diff
@@ -75,14 +75,17 @@ \section{Honors}
 \section{Education}
 
 \datedline{\textbf{Changshu Institute of Technology}, China}{Sep 2016 -- Jul 2020}
-  Bachelor of Engineering, Automotive Service Engineering. I also have passed the CET-6 at this period.
+\begin{itemize}
+  \item Bachelors of Engineering, Automotive Service Engineering
+  \item Completed the College English Test at the CET6 level
+  \item Competed in the university's autonomous driving vehicle competition
```

Comment:
Add more detail about the autonomous driving vehicle competition, was it just at your university, regional, national? Did you place, if high, you should mention that. What were your contributions to the team?

Reply:
The college lab I stayed before, attended the regional competition, but the place was not high. So, they quickly focused on the production things but they failed because they didn't want to invest more on independent research. They want to pack some open-source solution to sell out. The independent research infrastructures, I had mentioned in my final graduation project which is written in Chinese: https://m2team248-my.sharepoint.com/personal/mourinaruto_m2team248_onmicrosoft_com/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fmourinaruto%5Fm2team248%5Fonmicrosoft%5Fcom%2FDocuments%2F%E8%87%AA%E5%8A%A8%E9%A9%BE%E9%A9%B6%E4%BA%BA%E6%9C%BA%E4%BA%A4%E4%BA%92%E7%B3%BB%E7%BB%9F%E7%A0%94%E7%A9%B6%2Epdf&parent=%2Fpersonal%2Fmourinaruto%5Fm2team248%5Fonmicrosoft%5Fcom%2FDocuments&ga=1

Kenji Mouri

Reply:
Finally, I uploaded my graduation design paper to my website, and added some details for that in my resume.
https://github.com/MouriNaruto/MouriNaruto/commit/e98e6e42b5fad56dca549fbd4ce6725b49f5dcf6